cmd/soci/commands/index/list.go

```diff
@@ -85,29 +91,35 @@ var listCommand = cli.Command{
 		})
 
 		writer := tabwriter.NewWriter(os.Stdout, 8, 8, 4, ' ', 0)
-		writer.Write([]byte("DIGEST\tSIZE\tIMAGE REF\tPLATFORM\tORAS ARTIFACT\n"))
+		if !quiet {
```

Comment:
We should just have a different path at this level and not reuse any of the non-quiet path. The non-quiet path can output the same digest twice if there are 2 image refs for the same image digest. e.g.

```
$sudo ./out/soci-cli index list               
DIGEST                                                                     SIZE    IMAGE REF                           PLATFORM       
sha256:3971d8aed2a17f1ad265b4bb1ee3ec5f6bd4393a7ae7d9122173f3fc584364da    765     docker.io/library/busybox:latest    linux/amd64    
sha256:3971d8aed2a17f1ad265b4bb1ee3ec5f6bd4393a7ae7d9122173f3fc584364da    765     docker.io/library/busybox:v1        linux/amd64    
sha256:705af72f632dc283c4379efdd67e054f9cf6c21c302ddd25c07b65960bbdc581    765                                         linux/amd64
```